prefect_aws/__init__.py

```diff
@@ -6,6 +6,14 @@
 from .secrets_manager import AwsSecret
 from .workers import ECSWorker
 
+from prefect._internal.compatibility.deprecated import (
+    register_renamed_module,
+)
+
+register_renamed_module(
+    "prefect_aws.projects", "prefect_aws.deployments", start_date="Jun 2023"
```

Comment:
Will this display an automatically calculated deprecation end date of 6 months from June 2023?